student/models.py

```diff
@@ -59,6 +61,11 @@ class year, major, friends, etc. An object is only created for the
     first_name = models.CharField(max_length=255, default='', null=True)
     last_name = models.CharField(max_length=255, default='', null=True)
     disabilities = models.NullBooleanField(null=True, default=False)
+    advisors = models.ManyToManyField(Advisor, related_name='students')
+    primary_major = models.CharField(max_length=255, default='')
+    other_majors = ArrayField(models.CharField(max_length=255, default=''), default=list)
+    minors = ArrayField(models.CharField(max_length=255, default=''), default=list)
+    sections = models.ForeignKey(timetable_models.Section, null=True)
```

Comment:
maybe we want to name this something more descriptive? like Section is the model we are using bc it captures all the course information we need - maybe we want it to be called something like `sis_registered_courses` or `registered_courses` something like that.